src/coreclr/jit/emit.h

```diff
@@ -2793,6 +2794,58 @@ inline unsigned emitter::emitGetInsCIargs(instrDesc* id)
     }
 }
 
+//-----------------------------------------------------------------------------
+//  emitGetMemOpSize: Get the memory operand size of instrDesc.
+//
+//  Arguments:
+//       id - Instruction descriptor
+//
```

Comment:
Is it worth a remark that this exists because certain instructions have differing register and memory operand sizes?

e.g. `vextractf128` has a `128-bit` output (register or memory) but a `256-bit` input (register).
`vinsertf128` is the inverse with a `256-bit` output (register), a `256-bit` input (register), and a `128-bit` input (register or memory)
etc

Reply:
Sure, The method should also be under `#ifdef DEBUG` because it is only used in `emitDispIns()` and assert.